backend/tests/reservation/postSingle.js

```diff
@@ -26,7 +26,7 @@ describe('POST reservations/single', () => {
 
     const { reservationID, result } = response.body;
     assert.strictEqual(result, 'Added single reservation');
-    assert.isNumber(reservationID, 'Expected reservationID to be a number');
+    assert.isString(reservationID, 'Expected reservationID to be a number');
```

Comment:
Should the error message now say "Expected reservationID to be a string"? Same on line 170

Reply:
Good catch. Fixed